source/includes/harvest/_introduction.md

```diff
@@ -172,6 +172,7 @@ The timestamps below are Eastern Time.
 
 | Date                          | Description                                                                                                                       |
 |-------------------------------| --------------------------------------------------------------------------------------------------------------------------------- |
+| May 11, 2023 12:00:00PM | Added ability to update closed openings in the [Edit Openings Endpoint](#patch-edit-openings)
```

Comment:
I will change this to the date of the actual deploy. When the PR lands I'll push the documentation and change this date if necessary.